README.md

```diff
@@ -860,11 +873,84 @@ class PluginHost {
   }
 
   [Symbol.dispose]() {
+    this.#socket = undefined;
```

Comment:
Do these assignments do anything necessary to disposal, or are they just stylistic to keep something (what?) from referring to the private fields after Symbol.dispose is run?

Reply:
It's not necessary for the `Disposable` protocol, but good housekeeping in general. Any resources held by `#socket` would be freed when `#disposables` is disposed. Removing the reference to `#socket` would allow the object in `#socket` to be GC'd (since it's no longer usable). 

In addition, a disposable object should try to ensure that any access is consistent with its "disposed" state, and clearing `#socket` (and `#channel`) might be one way to ensure that any attempt to use `PluginHost` after it has been disposed will fail due to a `ReferenceError` being thrown from any attempt to use the undefined `#socket`.